viewer/packages/camera-manager/src/ComboControls.ts

```diff
@@ -418,12 +419,12 @@ export class ComboControls extends EventDispatcher {
     };
 
     const onMouseUp = () => {
-      window.removeEventListener('mousemove', onMouseMove);
-      window.removeEventListener('mouseup', onMouseUp);
+      window.removeEventListener('pointermove', onMouseMove);
+      window.removeEventListener('pointerup', onMouseUp);
     };
 
-    window.addEventListener('mousemove', onMouseMove, { passive: false });
-    window.addEventListener('mouseup', onMouseUp, { passive: false });
+    window.addEventListener('pointermove', onMouseMove, { passive: false });
+    window.addEventListener('pointerup', onMouseUp, { passive: false });
```

Comment:
What does this do?

Reply:
They are responsible for mouse move during rotation. Or if you mean why are they changed, it's because porting initial event to `pointerdown` requires everything else to have the same type of event to work properly.